lib/webrick-patch.rb

```diff
@@ -0,0 +1,36 @@
+require 'webrick/https'
+
+CIPHERS = 'AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256'
```

Comment:
Well, not exactly. If you take Mozilla's "Modern" configuration and remove ECDHE+DHE you are left with nothing. This is because they're optimizing for PFS, which is less relevant to us (but still would be nice to have).